configure/migrator/migrator.Job.yaml

```diff
@@ -19,7 +19,9 @@ spec:
       containers:
       - name: migrator
         image: "index.docker.io/sourcegraph/migrator:3.38.1@sha256:4bd4fd68fc3e6d1c947847aec126ff8c4577da80cbd11252c6c72a482722ad66"
-        args: ["up"]
+        args:
+          - up
+          - -db=all
```

Comment:
I'm undecided about whether we need to make this change, but if we do, shouldn't the frontend deployment also be updated?